### DISCUSSION TITLE
Is there an option in SEB to have it boot back to SEB if they reboot their device?

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-win-refactoring#898] [Q&A]
Hi,

I was just wondering if a user were to reboot their device while in SEB, upon rebooting the device would it be possible to bring them back to SEB immediately? I was wondering if there is any feature like that. If not, would there be any alternative solutions?

**Comment by dbuechel** [comment]
That is to my knowledge only working in SEB for iOS, but neither in SEB for Windows nor macOS. All SEB editions however will show a red lock screen when a student attempts to re-start an exam which has previously been aborted abnormally (i.e. by not using the quit button, keyboard shortcut, a quit link or SEB Server quit instruction).

**Reply by Nikolas-Dragolas** [reply]
Hi  dbuechel,
It is quite inconvenient and time-consuming (e.g., in case of a power outage or something similar) to enter the password on every student's computer, especially since we are using around 100 computers. In our case, we are not upgrading beyond version 3.4.1 precisely for this reason. It would be beneficial to have an option to disable this screen through the SEB Config Tool. Personally, I couldn't find such an option. If such an option exists, please share it.

**Reply by dbuechel** [reply]
> Implemented configuration option for session integrity verification (https://github.com/SafeExamBrowser/seb-win-refactoring/issues/790)
>
> \- https://github.com/SafeExamBrowser/seb-win-refactoring/releases/tag/v3.8.0

**Reply by Nikolas-Dragolas** [reply]
Thank you very much for the guidance and help!
